app/src/main/java/ru/chaichuk/hwapp/db/MoviesDataBase.kt

```diff
@@ -0,0 +1,24 @@
+package ru.chaichuk.hwapp.db
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import ru.chaichuk.hwapp.data.Movie
+
+@Database(entities = [Movie::class], version = 1)
+abstract class MoviesDataBase : RoomDatabase() {
+
+    abstract val moviesListDao : MoviesListDao
```

Comment:
that's a good idea, then the abstract method

Reply:
I copied this code from workshop. And I saw it in other examples. Is there something wrong with it?

Reply:
no no, I just wont to say that good job :)